components/editor/overview.md

```diff
@@ -86,15 +69,20 @@ The Telerik UI for Blazor Editor uses the ProseMirror engine and it depends on i
 
 The Blazor HTML Editor interacts with its content (value) like all standard components - through its `Value` parameter. You can use it to get and set the HTML string the editor will work with. You can read more about value binding and data binding [here]({%slug get-started-value-vs-data-binding%}).
 
-You can use the following features to get or set the editor content:
+#### You can use the following features to get or set the editor content:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* `@bind-Value` - the recommended approach of using two-way binding to get and set the content of the editor. It lets your view-model provide the initial value, and it will update the view-model as the user alters the HTML.
+| Parameter | Type and Default value | Description |
+|-----------|------------------------|-------------|
+| `@bind-Value`  | `EventCallback<string>` | The recommended approach of using two-way binding to get and set the content of the editor. It lets your view-model provide the initial value, and it will update the view-model as the user alters the HTML. |
+| `DebounceDelay`  | `int` <br /> `100ms` | The time in milliseconds that passes between updates on the `Value`. The default is `100ms` and if that causes performance issues with many repaints on your view, you can increase it. Since the editor is expected to handle longer editing sessions and larger content than regular inputs, we added this parameter to debounce the view-model updates and events. |
+| `ValueChanged`  | `EventCallback` | Allows you to receive the Editor value and act upon it. If you use the `ValueChanged` event (one-way binding), you can effectively cancel the user's input by not updating the view-model, or you can even alter it with something else. |
 
-* `DebounceDelay` - the time in milliseconds that passes between updates on the `Value`. The default is `100ms` and if that causes performance issues with many repaints on your view, you can increase it. Since the editor is expected to handle longer editing sessions and larger content than regular inputs, we added this parameter to debounce the view-model updates and events.
+## Validation
```

Comment:
The Validation section should be split into two parts. The part about SignalR message size should be similar to https://docs.telerik.com/blazor-ui/components/fileselect/overview#large-file-support and have its own heading.